src/operator/elementwise_binary_scalar_op.cc

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file elementwise_binary_op.cc
+ * \brief elementwise binary operator
+*/
+#include "./elementwise_binary_scalar_op-inl.h"
+
+namespace mxnet {
+namespace op {
+template<>
+Operator* CreateElementwiseBinaryScalarOp<cpu>(elembinary::ElementwiseBinaryScalarOpType type,
+                                               ScalarOpParam param) {
+    return CreateElementwiseBinaryScalarOp_<cpu>(type, param);
+}
+
+// DO_BIND_DISPATCH comes from static_operator_common.h
+template<typename ForwardOp>
+Operator* ElementwiseBinaryScalarOpProp<ForwardOp>::CreateOperator(Context ctx) const {
+  DO_BIND_DISPATCH(CreateElementwiseBinaryScalarOp, GetOpType<ForwardOp>(), param_);
+}
+
+DMLC_REGISTER_PARAMETER(ScalarOpParam);
+
+MXNET_REGISTER_OP_PROPERTY(_PlusScalar, ElementwiseBinaryScalarOpProp<mshadow::op::plus>)
+.describe("Perform an elementwise plus.")
+.add_argument("lhs", "Symbol", "Input data to activation function.")
```

Comment:
consider change lhs to array, as it is the array input.

Change description to input array operand to the operation